runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -513,8 +513,18 @@ public void requestFocus()
 			OSXUtil.requestFocus();
 		}
 
-		frame.requestFocus();
-		giveClientFocus();
+		if (System.getProperty("os.name").contains("10"))
```

Comment:
Is this really the best way to detect windows 10? Among other things, can't you at least do `OSType.getOSType() == OSType.Windows`?

What's the full string of `System.getProperty("os.name")` anyway? Seems like you should be matching its exact value instead of doing `.contains("10")`...

Reply:
The full string is "Windows 10", i will fix this.